test/integration_test/aws/run.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
+
+# wait localstash pod status to running
```

Comment:
We don't need to generate a temp file `pods.status`. The following command can be used to count the number of pods that are not running.  

```
kubectl get pods -n "${namespace}" ${labels} --no-headers | grep -v Running)
```

Reply:
done in https://github.com/chaos-mesh/chaos-mesh/pull/1477/commits/e39ff58e5a3f906734dd712c2e21ec87bcbc83e2